pkg/lib/authn/user/store.go

```diff
@@ -256,6 +289,8 @@ func (s *Store) QueryPage(listOption ListOptions, pageArgs graphqlutil.PageArgs)
 	if err != nil {
 		return nil, 0, err
 	}
+	stmt, args, err := query.ToSql()
+	fmt.Println("tung DEBUG SQL", stmt, args, err)
```

Comment:
Remove debug log. Consider adding debug print to SQLExecutor. So that when LOG_LEVEL=debug, the SQL got printed when executed.

Reply:
I've removed the debug log in later commits. There might be too much noise when `LOG_LEVEL=debug` if we print all sqls? Prefer keep the current behavior or add another config for printing sqls.

Reply:
OK.